velox/functions/prestosql/Bitwise.h

```diff
@@ -28,6 +30,28 @@ struct BitwiseAndFunction {
   }
 };
 
+template <typename T>
+struct BitCountFunction {
+  FOLLY_ALWAYS_INLINE bool call(int64_t& result, int64_t num, int32_t bits) {
+    VELOX_USER_CHECK(
+        (bits >= 2 && bits <= kMaxBits),
+        "Bits specified in bit_count must be between 2 and 64,"
+        " got {}",
```

Comment:
any particular reason to put " got {}" on a new line; it will fit on the previous line just fine